kube/base/configmap.yaml

```diff
@@ -0,0 +1,217 @@
+---
```

Comment:
Do we need this file now that we have the `configMapGenerator` definited in the `kustomization.yaml`?

Reply:
That's a good point, I think it's safe to delete it now. Update coming shortly with the other changes :+1: